### PR TITLE
[gui] add container.showtitle

### PR DESCRIPTION
**Pull request by phil65**
Useful esp. for breadcrumbs, as discussed somewhen with @BigNoid  .
@xhaggi

### CODE REVIEW

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -474,7 +474,8 @@ const infomap mediacontainer[] = {{ "hasfiles",         CONTAINER_HASFILES },
                                   { "hasthumb",         CONTAINER_HAS_THUMB },
                                   { "sortmethod",       CONTAINER_SORT_METHOD },
                                   { "sortorder",        CONTAINER_SORT_ORDER },
-                                  { "showplot",         CONTAINER_SHOWPLOT }};
+                                  { "showplot",         CONTAINER_SHOWPLOT },
+                                  { "showtitle",         CONTAINER_SHOWTITLE }};
```
Comment (xhaggi) [inline xbmc/GUIInfoManager.cpp]: indentation

File: xbmc/GUIInfoManager.cpp [hunk 2]
```diff
@@ -2094,10 +2095,14 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
     return GetMultiInfoLabel(GUIInfo(info), contextWindow);
     break;
   case CONTAINER_SHOWPLOT:
+  case CONTAINER_SHOWTITLE:
     {
       CGUIWindow *window = GetWindowWithCondition(contextWindow, WINDOW_CONDITION_IS_MEDIA_WINDOW);
       if (window)
```
Comment (xhaggi) [inline xbmc/GUIInfoManager.cpp]: enclosing braces

Comment (phil65) [inline xbmc/GUIInfoManager.cpp]: oops.

**Comment by phil65** [comment]
@xhaggi 	updated, thx for checkin.

**Comment by xhaggi** [comment]
No problem

**Comment by phil65** [comment]
jenkins build this please.

**Pull request merged**